lib/hooks/usePrizePoolAddress.js

```diff
@@ -0,0 +1,22 @@
+import { POOL_ALIASES } from 'lib/constants'
+import { useRouter } from 'next/router'
+
+export const usePrizePooladdress = () => {
+  const router = useRouter()
+
+  const poolAlias = Array.isArray(router.query.poolAlias)
```

Comment:
When is poolAlias an array? Shouldn't query params typically all be strings?

Reply:
I'm not quite sure.. 
Typehints for the `next/router` were saying it could possibly be an array, so I figured I'd handle that better. But I don't know how to format the URL to get that case.

Reply:
Yeah sometimes I'm not sure what's up with those hints. 🙄